pkg/kube/inject/template.go

```diff
@@ -195,12 +196,25 @@ func appendMultusNetwork(existingValue, istioCniNetwork string) string {
 			log.Warnf("Unable to unmarshal Multus Network annotation JSON value: %v", err)
 			return existingValue
 		}
+		istioCniNetworkNs, istioCniNetworkName := kstrings.SplitQualifiedName(istioCniNetwork)
```

Comment:
I think it worth adding unit tests to have coverage for this chage

Reply:
what case is missing? for me the testdata looks like it covers exactly what you're describing in the issue